server/src/internalClusterTest/java/org/elasticsearch/snapshots/GetSnapshotsIT.java

```diff
@@ -451,6 +447,189 @@ public void testFilterBySLMPolicy() throws Exception {
         assertThat(getAllSnapshotsForPolicies(GetSnapshotsRequest.NO_POLICY_PATTERN, "*"), is(allSnapshots));
     }
 
+    public void testSortAfterStartTime() {
```

Comment:
I did not explicitly add tests for all columns as they use the same functionality and limited myself to those that I needed for full coverage only here.